packages/nimble-components/src/rich-text/editor/models/create-tiptap-editor.ts

```diff
@@ -195,13 +212,41 @@ function createCustomMentionExtension(
             pluginKey: new PluginKey(config.key),
             allowSpaces: true,
             render: () => {
+                let inSuggestionMode = false;
                 return {
                     onStart: (props): void => {
+                        inSuggestionMode = true;
                         config.mentionUpdateEmitter(props.query);
+                        activeMentionCharacterEmitter(props.text.slice(0, 1));
+                        activeMentionCommandEmitter(props.command);
+                        mentionListBox?.show({
+                            filter: props.query,
+                            anchorNode: props.decorationNode as HTMLElement
+                        });
                     },
-
                     onUpdate: (props): void => {
+                        if (!inSuggestionMode) {
+                            return;
+                        }
                         config.mentionUpdateEmitter(props.query);
+                        activeMentionCommandEmitter(props.command);
+                        mentionListBox?.show({
+                            filter: props.query,
+                            anchorNode: props.decorationNode as HTMLElement
+                        });
+                    },
+                    onKeyDown: (props): boolean => {
```

Comment:
Should this handler also be returning early if `!inSuggestionMode`? Why do we need to call the mention listbox key handling code in that case?

Reply:
Added `!inSuggestionMode` case to return early in `onKeyDown` when `inSuggestionMode` is in false.